build.gradle

```diff
@@ -33,6 +33,17 @@ dependencies {
     compile 'jfree:jfreechart:1.0.13'
 }
 
+project(':fakeRobot') {
```

Comment:
Please update the WPILibVersion plugin while you do this. It should look like this: https://github.com/wpilibsuite/allwpilib/blob/master/build.gradle#L3-L12